framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/model/components/CommonConfig.java

```diff
@@ -11,7 +11,7 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
-*/
+ */
```

Comment:
Not sure why, but my formatter adds an extra space here. If this is important, can you try in your IDE? Not sure if I have something different from your configuration.